### ISSUE TITLE
pyscripter fail

### ISSUE THREAD

**Issue by ololoe** [Drekin/win-unicode-console#21]
import win_unicode_console
    win_unicode_console.enable()
    
    print "ÄäБб"

If run from pyscripter

    Message	File Name	Line	Position	
    Traceback				
        <module>	file.py	2		
        enable	D:\Program Files\programming\Python27\lib\site-packages\win_unicode_console\__init__.py	24		
        enable	D:\Program Files\programming\Python27\lib\site-packages\win_unicode_console\streams.py	304		
        check_stream	D:\Program Files\programming\Python27\lib\site-packages\win_unicode_console\streams.py	273		
        __getattr__	<string>	523		
        __getattr__	D:\Program Files\programming\PyScripter\Lib\rpyc.zip\rpyc\core\netref.py	150		
        syncreq	D:\Program Files\programming\PyScripter\Lib\rpyc.zip\rpyc\core\netref.py	71		
        sync_request	D:\Program Files\programming\PyScripter\Lib\rpyc.zip\rpyc\core\protocol.py	434		
    AttributeError: DebugOutput instance has no attribute 'fileno'

**Comment by Drekin** [comment]
It seems that `PyScripter` uses its own stream objects having no `fileno` attribute, and accessing it raises `AttributeError` rather than expected `io.UnsupportedOperation`. A quick fix would be to add `AttributeError` to the `except` clause. However it is not clear, how `win-unicode-console` should interact with `PyScripter` custom streams. If it replaces them, we loose `PyScripter` functionality. If it doesn't, we don't have Unicode IO.

**Comment by ololoe** [comment]
Yeah, workarounds are not compatible to each other. Is there a chance you'll find a workaround together with pyscripter developers (if they are still alive)?

**Comment by Drekin** [comment]
It depends on how `PyScripter` stream objects work and what they do. If they are basically wrappers over the standard streams (this is how `colorama` works, it wraps the strandard streams and filters and interprets ANSI constrol sequences) the it would be enough to let `PyScripter` wrap `win-unicode-console` streams instead of the standard ones – as is done in the `colorama` case.
